persona.proto

```diff
@@ -0,0 +1,32 @@
+syntax = "proto3";
+
+// This protobuf is for adding public/private key pairs to the messaging.
+message persona {
+    /*
+    *   Unique Local Name
+    */
+    string local_name = 1;
+    string mac_address = 2;
+    int32 node_num = 3;
+
+    /*
+    *   Self's public and private key
+    */
+    bytes public_key = 4;
+
+    bytes private_key = 5;
+
+    message all_peers {
+        repeated Peer peer = 1;
+    }
+}
+
+message secret_book{
+    repeated persona person = 1;
+}
+
+message Peer {
```

Comment:
A separate Peer protobuf message is not required if a boolean field is added to Persona indicating if it is self or peer

Reply:
Removed peer and added as boolean field to Persona.